### DISCUSSION TITLE
Loading openbrush glb2 files

### DISCUSSION THREAD

**Discussion by arpu** [donmccurdy/glTF-Transform#419] [Q&A] [answered]
Hi,

i get some errors on loading openbrush glb2 files https://icosa-gallery.github.io/icosa-viewer/formats/gltf2/all_brushes.glb
error: ENOENT: no such file or directory, open '/tmp/https:/www.tiltbrush.com/shaders/brushes/OilPaint-f72ec0e7-a844-4e38-82e3-140c44772699/OilPaint-f72ec0e7-a844-4e38-82e3-140c44772699-v10.0-MainTex.png'

would it be possible to avoid the error and still to load the glb for modification

**Comment by arpu** [comment, marked as answer]
looks like this is not standard confirm and best solution is to export the model with included Textures in openBrush

**Reply by donmccurdy** [reply]
Yeah, `gltf-pipeline` chokes on this GLB too, and `gltfpack` will run but omits all the textures. For security reasons I don't think any of these tools want to make network requests by default, and the files are at absolute URLs on the web.

In a script it would be possible to load the GLB, download the images first, update the image URIs in the GLB, and then continue processing with glTF-Transform. But even then there's the `GOOGLE_tilt_brush_material` extension present, so this isn't going to render with all the shader effects in any standard glTF viewer. Instead you'd get something like:

![Screen Shot 2021-11-09 at 10 24 55 AM](https://user-images.githubusercontent.com/1848368/140982778-e669430c-9d0c-4f64-bd86-60288311ce35.png)

**Reply by arpu** [reply]
looks like https://github.com/icosa-gallery/icosa-viewer  has done some work for this

**Reply by arpu** [reply]
newer version from openbrush includes the brush Textures so we do not need the extra Texture url download
